app/lib/shared/handler_helpers.dart

```diff
@@ -7,11 +7,21 @@ import 'dart:io';
 
 import 'package:appengine/appengine.dart';
 import 'package:logging/logging.dart';
+import 'package:path/path.dart' as path;
 import 'package:shelf/shelf.dart' as shelf;
 import 'package:shelf/shelf_io.dart' as shelf_io;
 
-Future runHandler(Logger logger, shelf.Handler handler, {bool shared: false}) {
+import '../frontend/service_utils.dart';
+
+import 'utils.dart' show fileAnIssueContent;
+
+Future runHandler(Logger logger, shelf.Handler handler,
+    {bool sanitize: false, bool shared: false}) {
   handler = redirectToHttpsWrapper(handler);
```

Comment:
Are these applied in the right order? Ie. should we redirect to https before parsing auth/sanitizing the request?

Reply:
I don't see issues either way, reordered.